docs/integrations/sources/airtable.md

```diff
@@ -35,6 +33,7 @@ See information about rate limits [here](https://support.airtable.com/hc/en-us/a
 
 | Version | Date       | Pull Request                                             | Subject                                  |
 | :------ | :--------- | :------------------------------------------------------- | :--------------------------------------- |
+| 0.1.4   | 2022-12-22 | [20846](https://github.com/airbytehq/airbyte/pull/20846) | Migrated to Metadata API for dynamic schema generation           |
```

Comment:
Can you share more about the migration plan for users?  I think this is at least a minor Semver change as a config value was removed... but might this be a major version change?

For users who previously provided the `base_id` and `tables` configuration values, will they need to re-configure the connector, or will things work as they used to as the new base+table streams will be unselected?

Reply:
@evantahler 

The Customers should re-do the sync and ideally re-set up the connector with all the data. Here are a couple of reasons why:
1) the schemas were declared incorrectly, therefore even if present customers are using the connector, they might have data issues because the discovery  mechanism was not working correctly
2) It could be the major version, but I don't see a big difference, if you insist  - let's go with it.
3) the previous configuration should also work, BTW, but there could the issues with normalization, if applicable. Therefore:
The users should re-configure their connection, for only 1 valid reason  - to have the schemas and data fulfilled correctly.

Reply:
@evantahler 
the version was updated to `0.2.0`  to reflect the major version updates.